src/Components/CreateImageWizardV2/steps/Review/ReviewStepTables.tsx

```diff
@@ -220,6 +223,19 @@ export const PackagesTable = () => {
                 </Td>
               </Tr>
             ))}
+            {groups.map((grp, grpIndex) => (
+              <Tr key={grpIndex}>
+                <Td className="pf-m-width-30">@{grp.name}</Td>
+                <Td>{grp.description ? grp.description : 'Not available'}</Td>
+                <Td className="pf-m-width-30">
+                  {grp.repository === 'distro'
```

Comment:
Recommended package groups have "Not available" in the repo column

![image](https://github.com/osbuild/image-builder-frontend/assets/49452678/2e34e3e1-393d-477f-8b27-186074f9d1ca)